act-rules-format.bs

```diff
@@ -451,63 +496,67 @@ Definitions {#definitions}
-    <p>A common example of accessibility requirements are the WCAG 2.1 success criteria. There are other standards, including W3C standards that include recommendations for accessibility, such as WAI-ARIA and HTML. Accessibility requirements are also often found in company policies, regional standards or in legislation.</p>
+    <p>A common example of accessibility requirements are the WCAG 2.1 success criteria. There are other standards, including W3C standards that have recommendations for accessibility, such as WAI-ARIA and HTML. Accessibility requirements are also often found in company policies, regional standards or in legislation.</p>
   </dd>
 
   <dt><dfn>Accessibility requirements document</dfn><dt>
   <dd>
-    <p>A document, such as a standard, contract, policy or regulation, that includes [=accessibility requiremenst=]. For example, WCAG 2.1, WAI-ARIA 1.1, HTML 5.2, BBC HTML Accessibility Standards v2.0</p>
+    <p>A document, such as a standard, contract, policy or regulation, that includes [=accessibility requirements=]. For example, WCAG 2.1, WAI-ARIA 1.1, HTML 5.2, BBC HTML Accessibility Standards v2.0</p>
+  </dd>
+
+  <dt><dfn>Outcome</dfn></dt>
+  <dd>
+    <p>One of three types of conclusions that come from evaluating an ACT Rule on a [=test subject=]. An outcome can be:<p>
+    <ul>
+      <li>**Inapplicable:** No part of the test subject matches the applicability</li>
+      <li>**Passed:** The [=test target=] meets all expectations</li>
+      <li>**Failed:** The [=test target=] does not meet all expectations</li>
+    </ul>
+    <div class="note">
+      <p>**Note:** While *inapplicable* is a valid outcome for ACT Rules, it might not be a valid result for all [=accessibility requirements=]. Notably the success criteria of WCAG 2.0 and WCAG 2.1 can only be evaluated to satisfied (passed and inapplicable) or not satisfied (failed).</p>
+    </div>
+    <div class="note">
+      <p>**Note:** A rule has one passed or failed outcome for every [=test target=]. When there are no test targets the rule has one inapplicable outcome. This means that each [=test subject=] will have one or more [=outcomes=].</p>
+    </div>
+    <div class="note">
+      <p>**Note:** Implementers using the [[EARL10-Schema]] can express the outcome with the [outcome property](https://www.w3.org/TR/EARL10-Schema/#outcome). In addition to `passed` `failed` and `inapplicable`, EARL 1.0 also defined an `incomplete` outcome. While this cannot be the outcome of an ACT Rule when applied in its entirety, it often happens that rules are only partially evaluated. For example, when applicability was automated, but the expectations have to be evaluated manually. Such "interim" results can be expressed with the `incomplete` outcome.
```

Comment:
[Editorial]

Missing a comma in "In addition to `passed` `failed` and `inapplicable`,"